addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/welcome/NAWelcomeCameraHandler.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.handler.welcome;
+
+import static org.openhab.binding.netatmo.NetatmoBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.netatmo.config.NetatmoWelcomeConfiguration;
+
+import io.swagger.client.model.NAWelcomeCameras;
+
+/**
+ * {@link NAWelcomeCameraHandler} is the class used to handle the Welcome Camera Data
+ *
+ * @author Ing. Peter Weiss - Welcome camera implementation
+ *
+ */
+public class NAWelcomeCameraHandler extends AbstractNetatmoWelcomeHandler {
+
+    private NetatmoWelcomeConfiguration configuration;
```

Comment:
There are many informations in the class AbstractNetatmoThingHandler which will not be used in the welcome things, like signalTresholds, measuredChannels an measures.
If the Abstract class just holds all generic informations for all netatmo things, it would be a good idea to inherit from there.